go/tools/gazelle/gazelle/main.go

```diff
@@ -32,7 +32,7 @@ import (
 var (
 	goPrefix = flag.String("go_prefix", "", "go_prefix of the target workspace")
 	repoRoot = flag.String("repo_root", "", "path to a directory which corresponds to go_prefix, otherwise gazelle searches for it.")
-	mode     = flag.String("mode", "print", "print, fix or diff")
+	mode     = flag.String("mode", "fix", "print, fix or diff")
```

Comment:
Could you also expand the "print, fix or diff" to explain a bit more what each one does?

I guess fix fix the exiting BUILD file, print output the generated build file and diff shows a diff between the generated and the existing (or is it a diff corresponding to what fix do?)